python/gauss2d/fit/parameters.h

```diff
@@ -34,8 +34,9 @@
 
 #include "gauss2d/fit/parameters.h"
 #include "gauss2d/fit/transforms.h"
-#include "parameters/limits.h"
-#include "parameters/transform.h"
+
+#include <modelfit_parameters/limits.h>
+#include <modelfit_parameters/transform.h>
```

Comment:
In addition to my earlier comment about `<>`, please [include dependencies before local includes](https://developer.lsst.io/cpp/style.html#include-statements-should-be-sorted-and-grouped).

Reply:
You mean:
```#include "lsst/modelfit/parameters.h"

#include "gauss2d/fit/parameters.h"```

... separating them because the latter file is part of this package, though I still need to include it by the full namespace?